openmdao/api.py

```diff
@@ -112,6 +112,9 @@
     MPIWarning, UnitsWarning, SolverWarning, OMDeprecationWarning, \
     OMInvalidCheckDerivativesOptionsWarning
 
+# Utils
+from openmdao.utils.file_utils import image2html, text2html
+
```

Comment:
Were these added to api intentionally? Will people use these?

Reply:
The idea was that if someone is writing their own reports, they could use these functions to easily wrap their text output or image files. I put this in here when I as adding dymos reports since dymos was already importing openmdao.api, but it's not a big deal to move it out of here if it doesn't seem like it belongs.